src/select-list.c

```diff
@@ -0,0 +1,157 @@
+/*   string/int GtkListStore for GtkComboBox (display name plus value)
```

Comment:
Please format this with `clang-format`. There are only minor findings, but still it would be great to have this in `dev`  clean.

Reply:
Done